src/helpers/configuration.sh

```diff
@@ -20,30 +20,17 @@ if [[ $@ =~ --bypass-automatic-configuration ]]; then
   return 0
 fi
 
-
-# Ask and store main-branch-name, if it isn't known yet.
-if [[ -z "$main_branch_name" ]]; then
-  echo "Please enter the name of the main dev branch (typically 'master' or 'development'):"
-  read main_branch_name
-  if [[ -z "$main_branch_name" ]]; then
-    echo_error_header
-    echo_error "You have not provided the name for the main branch."
-    echo_error "This information is necessary to run this script."
-    echo_error "Please try again."
-    exit_with_error
+# Show configuration prompt
+if [[ "$(is_git_town_configured)" == false ]]; then
+  echo "Git Town hasn't been configured for this repository."
+  echo "Please run 'git town config --setup'."
+  echo "Would you like to do that now? y/n"
+
+  read setup_git_town_now
+  if [[ "$setup_git_town_now" == y ]]; then
```

Comment:
@charlierudolph has previously recommended not quoting strings unless necessary (I still think it looks weird), but it works. Same as `if [ ... == true ] ...`